editor/project_manager.cpp

```diff
@@ -1042,10 +1042,18 @@ void ProjectListItemControl::set_project_icon(const Ref<Texture2D> &p_icon) {
 	project_icon->set_texture(p_icon);
 }
 
+bool _project_feature_looks_like_version(const String &p_feature) {
+	return p_feature.contains(".") && p_feature.substr(0, 3).is_numeric();
```

Comment:
This code will return true even for versions like `123456.0` and `4.0.1`, and will return false for commit hashes like `12345678`.

Reply:
I mean, we could use a more restrictive regex and check for `[34]\.[0-9]+`, but that would probably be an overkill. Good enough 👍

Reply:
Well we want it to recognize Godot 5.0, 6.2, 7.4, etc, not just `[34]`.

Reply:
No, we don't want that at the moment, so there is no reason to make such a requirement. Of course, it would be kinda easy to miss updating this in 10 years when Godot 5.0 is out, but a code comment could solve that.